README.md

```diff
@@ -9,15 +9,15 @@ Design and proposal can be found here: http://bit.ly/k8s-LWS.
 
 ## Feature overview
 
-- **Group of Pods as a unit:** A tightly managed group of pods that represent a “super pod” with the requirements
+- **Group of Pods as a unit:** Supports a tightly managed group of pods that represent a “super pod”
   - **Unique pod identity:** Each pod in the group has a unique index from 0 to n-1.
   - **Parallel creation:** Pods in the group will have the same lifecycle and be created in parallel.
-- **Dual-template, one for leader and one for the workers:** The group is composed of a single leader and a set of workers, and the api should allow to specify a template for the workers and optionally use a second one for the leader pod.
-- **Multiple groups with identical specifications:** We should be able to create multiple “replicas” of the above mentioned group. Each group is a single unit for rolling update, scaling, and maps to a single exclusive topology for placement. 
-- **A scale subresource:** We will expose a scale endpoint for HPA to dynamically scale the number replicas (aka number of groups)
-- **Rollout and Rolling update:** It will be performed at the group level, which means we upgrade the groups one by one as a unit (i.e. the pods within a group are updated together).
-- **Topology-aware placement:** Be able to ensure that pods in the same group will be co-located in the same topology.
-- **All-or-nothing restart for failure handling:** Certain ML inference stacks require all pods in the group to be recreated if one pod in the group failed or one container in the pods is restarted.
+- **Dual-template, one for leader and one for the workers:** Supports pod group that is composed of a single leader and a set of workers, and allow to specify a template for the workers and optionally use a second one for the leader pod.
```

Comment:
```suggestion
- **Dual-template, one for leader and one for the workers:** Pod group is composed of a single leader and a set of workers, and allow to specify a template for the workers and optionally use a second one for the leader pod.
```

Reply:
I would actually say: "A replica is a group of a single leader and a ...."

Reply:
updated